l2-contracts/contracts/bridge/L2StandardERC20.sol

```diff
@@ -158,29 +167,29 @@ contract L2StandardERC20 is ERC20PermitUpgradeable, IL2StandardToken, ERC1967Upg
 
     function name() public view override returns (string memory) {
         // If method is not available, behave like a token that does not implement this method - revert on call.
-        if (availableGetters.ignoreName) revert();
+        if (availableGetters.ignoreName) revert Unimplemented();
         return super.name();
     }
 
     function symbol() public view override returns (string memory) {
         // If method is not available, behave like a token that does not implement this method - revert on call.
-        if (availableGetters.ignoreSymbol) revert();
+        if (availableGetters.ignoreSymbol) revert Unimplemented();
         return super.symbol();
     }
 
     function decimals() public view override returns (uint8) {
         // If method is not available, behave like a token that does not implement this method - revert on call.
-        if (availableGetters.ignoreDecimals) revert();
+        if (availableGetters.ignoreDecimals) revert Unimplemented();
```

Comment:
Same here

Reply:
fixed it in https://github.com/matter-labs/era-contracts/pull/505